File: LeetCode/Jun19.java
//58. Length of Last Word

class Solution {
    public int lengthOfLastWord(String s) {
        s=s.trim(); int n=0;
        for(int i=s.length()-1;i>=0;i--)
        {
            char ch=s.charAt(i);
            if(ch!=' ') n++;
            else return n;
        }
        return n ;
    }
}

File: LeetCode/Jun19.cpp
//58. Length of Last Word
#include<string>
using namespace std;

class Solution {
public:
    int lengthOfLastWord(string s) 
    {
        int n=0;
        for(int i=s.length()-1;i>=0;i--)
        {
            if(s[i]!=' ') n++;
            if(s[i]==' ' && n>0) return n;
        }
        return n ;
    }
};